index.bs

```diff
@@ -1018,6 +1015,11 @@ Composition of the Technical Architecture Group</h5>
 	and <em class=rfc2119>may</em> be run at other times when initiated by the current chairs or the Team,
 	for example if a chair steps down or if a minority of the participants make such a request.
 
+	As <dfn export>Director Emeritus</dfn>,
+	Tim Berners-Lee has a ongoing standing invitation
+	to all regular sessions of the Technical Architecture Group,
```

Comment:
```suggestion
	to participate in all regular sessions and asynchronous activities of the Technical Architecture Group,
```
@nigelmegitt, Would this address https://github.com/w3c/process/pull/791#issuecomment-1761226783?